src/markup/patterns/components/defaults/collection.yaml

```diff
@@ -0,0 +1,2 @@
+notes: |
```

Comment:
This collection shows all of the classes to apply "default" styling to elements. Though I think this is pretty confusing if you browse to this page before you browse to it before you understand the whole `:not([class])` concept.

I'd like to beef up this intro to feel more like our "Documentation" pages. That way we can introduce the concept, _then_ show all the available relevant components.

Reply:
Do you have ideas on what that would look like? Is there a card for that?

Reply:
Most of the work would be figuring out what it should look like. I'm thinking we'd step through  concepts in the same order you might encounter them while writing code.

1. Show that elements have styles by default
2. Show that adding a class removes that default styling
3. Show that we have components to put that default styling back in place if you needed

_Maybe_ we can convey this as a series of patterns in a collection? But I think it'd be easier to understand if it flows like a tutorial.

It'd also be nice to explain why this stuff is in place. (So we don't need to manually reset styles all the time)